### PR TITLE
Go camel and documentation

### PR DESCRIPTION
**Pull request by kealan**

### CODE REVIEW

File: wrappers/go/mpin.go.in [hunk 1]
```diff
@@ -35,24 +35,26 @@ import (
...
+const G2S = 4 * PFS
+const GTS = 12 * PFS
 
 //  Hash function choice
 const SHA256 int = 32
 const SHA384 int = 48
 const SHA512 int = 64
 
-func OCT_free(valOctet *C.octet) {
+// Free memory associated with octet
```
Comment (nicolaasuni) [inline wrappers/go/mpin.go.in]: When commenting an exported function, golang requires to start the comment with the function name. In this case:
// OctetFree frees the memory associated with octet

This is also valid for the following functions.

File: wrappers/go/mpin.go.in [hunk 2]
```diff
@@ -121,656 +126,1100 @@ func OCT_toHex(valOctet *C.octet) string {
...
 
-/* return time in slots since epoch */
-func MPIN_today() int {
-	date := C.MPIN_today()
-	return int(date)
+/*
+
+Today's date as days elapsed from the epoch. This function uses the system clock
+
+Args:
```
Comment (nicolaasuni) [inline wrappers/go/mpin.go.in]: This is probably not required...

Comment (kealan) [inline wrappers/go/mpin.go.in]: I agree that you have golang functions to do this but there is no harm in have this function. Also there was a time in the past when we wanted to change the time slot period for days to hours. This function would mean that we would only have to do this at the C level and the python and golang would still work.

File: wrappers/go/mpin.go.in (hunk 1 above)
Comment (kealan) [inline wrappers/go/mpin.go.in]: OK I will fix this

File: wrappers/go/mpin.go.in (hunk 2 above)
Comment (nicolaasuni) [inline wrappers/go/mpin.go.in]: Sorry, I was referring to the empty comments, not the function :-)

**Pull request merged**